lib/lightning/projects.ex

```diff
@@ -145,6 +145,16 @@ defmodule Lightning.Projects do
     |> Repo.all()
   end
 
+  @spec select_project_for_user(user :: User.t()) :: Project.t()
```

Comment:
This specs return should be `... :: Project.t() | nil`, since they may not have one.

Perhaps it should be named `first_project_for_user` as well, since the name doesn't reflect the `limit` clause.